text/0000-no-more-extern-crate.md

```diff
@@ -70,29 +85,21 @@ For example, `cargo build`-ing a crate `my_crate` with a dependency on `rand`
 results in a call to rustc that looks something like
 `rustc --crate-name mycrate src/main.rs --extern rand=/path/to/librand.rlib ...`.
 
-When an external crate is specified this way,
-the crate will automatically brought into scope as if an
-`extern crate name_of_crate;`
-declaration had been added to the current crate root.
-This behavior won't occur when including a library using the `-l`
+When an external crate is specified this way, it will be automatically
+available to any module in the current crate through `use` statements or
+absolute paths (e.g. `::rand::random()`). It will _not_ be automatically
+imported at root level as happens with current `extern crate`.
```

Comment:
To expand on @petrochenkov's dislike (correct me if I'm wrong): if absolute paths work, relative paths at the root level pretty much *have to* work for the system to be self-consistent.

That is, `use foo::bar;` and `::foo::bar` *look for a top-level definition `foo`*.
This is *exactly* what happens when you write `foo::bar` in the root module.

If you hook "top-level definitions with name found in the list of `--extern` crates", they will *both* work identically. Otherwise, it would have to be a custom hack in the name resolution algorithm, and eventually, spceifically.